apps/deploy-web/next.config.js

```diff
@@ -10,6 +10,16 @@ const withPWA = require("next-pwa")({
 });
 const { withSentryConfig } = require("@sentry/nextjs");
 
+try {
+  const { browserEnvSchema } = require("./env-config.schema");
```

Comment:
this file is compiled as a part of a `build` npm script. So it is expected to only be present at docker build time. Or anytime the command is executed off